src/data/data.js

```diff
@@ -295,46 +257,60 @@ const pageData = {
       },
     ],
   },
-  navigation: {
-    items: [
-      {
-        text: 'Nosotros',
-        href: 'nosotros',
-      },
-      {
-        text: 'Contexto',
-        href: 'contexto',
-      },
-      {
-        text: 'Iniciativa',
-        href: 'iniciativa',
-      },
-      {
-        text: 'Registro',
-        href: 'registro',
-      },
-    ],
-    socialText: 'Encuentranos en nuestras redes sociales',
-    socialLinks: [
-      {
-        href: 'instagram.com',
-        icon: 'instagram',
-      },
+  common: {
+    initalSections: [
       {
```

Comment:
Could you please add the `header` component info before the navigation, as it is also part of the layout and it is present in every page?